### PR TITLE
Replaced ListItem component with StudentCard component

### PR DESCRIPTION
**Pull request by zyadelgohary**
I replaced the ListItem component with the StudentCard component in the "ClassMainScreen.js" file & changed the parameters so they fit with the StudentCard parameters.

### CODE REVIEW

File: Dev/QConnect/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js [hunk 1]
```diff
@@ -1,36 +1,47 @@
...
-
-export default ClassMainScreen;
+import React from 'react';
+import {View} from 'react-native';
+import {StudentCard} from 'components/StudentCard'
+import {Picture} from 'assets/splash.png'
+
+class ClassMainScreen extends React.Component {
+    state = {
+        students: [
```
Comment (awsm36z) [inline Dev/QConnect/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js]: Remove
TL;DR remove lines 9-20
               {
                name: "Khalid",
                avatar: {Picture},
                currentAssignment: "Al Baqara Page 1"
            },
            {
                name: "Zyad",
                avatar: {Picture},
                currentAssignment: "Al Nas Ayah 3"
            }
        ]
    };

because our student data will be coming from the class.json file.

File: Dev/QConnect/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js [hunk 2]
```diff
@@ -1,36 +1,47 @@
...
+            {
+                name: "Zyad",
+                avatar: {Picture},
+                currentAssignment: "Al Nas Ayah 3"
+            }
+        ]
+    };
+
+    componentDidMount() {
+        const students = require('model/class.json').students.map(s => ({
```
Comment (awsm36z) [inline Dev/QConnect/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js]: This is where the list of students actually comes from. No need for you to define your own students data.

File: Dev/QConnect/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js (hunk 1 above)
Comment (zyadelgohary) [reply Dev/QConnect/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js]: Ah, good catch. I was using it to test that the code actually works but forgot to remove it afterwards. Look at it it now, it should be updated isA.
